apps/aecore/lib/aecore/chain/block_validation.ex

```diff
@@ -121,7 +121,7 @@ defmodule Aecore.Chain.BlockValidation do
 
   @spec block_header_hash(Header.t()) :: binary()
   def block_header_hash(%Header{} = header) do
-    block_header_bin = Serialization.pack_binary(header)
+    block_header_bin = Header.header_to_binary(header)
```

Comment:
why is this Header.header_to_binary and in other things we use the Serialization module directly?